src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/VerifyWebSockets.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.ServiceModel;
+
+namespace WcfService
+{
+    [ServiceBehavior(InstanceContextMode = InstanceContextMode.PerSession, AddressFilterMode = AddressFilterMode.Any)]
+    public class VerifyWebSockets : IVerifyWebSockets
+    {
+        private String successMessage = "";
+
+        // This operation cannot return anything so in order to validate that it was called set the InstanceContextMode
+        //  to PerSession and set a variable that we can get with a call to the ValidateWebSocketsUsed operation.
+        public void ForceWebSocketsUse()
+        {
+            successMessage = "WebSocketsWasUsed";
```

Comment:
Nit: not required to fix -- you could have just set a 'bool' field if this was called and had ValidateWebSocketsUsed could return the bool.  That would eliminate the need for a magic string duplicated in both server and test code.